src/test/TxTests.h

```diff
@@ -255,6 +256,8 @@ Asset makeNativeAsset();
 Asset makeInvalidAsset();
 Asset makeAsset(SecretKey const& issuer, std::string const& code);
 Asset makeAssetAlphanum12(SecretKey const& issuer, std::string const& code);
+ChangeTrustAsset makePoolShareAsset(Asset const& assetA, Asset const& assetB,
```

Comment:
Nitpick: not crazy about this name because it doesn't make it clear whether it is the hashed one or the raw one.

Reply:
Updated to `makeChangeTrustAssetPoolShare`.

Reply:
I forgot to run `make format`...

Reply:
fixed